dotnet/Simple.Sauce/SauceOptions.cs

```diff
@@ -56,7 +153,92 @@ public SauceOptions()
         public string TimeZone { get; set; }
         public string TunnelIdentifier { get; set; }
         public bool VideoUploadOnPass { get; set; }
+        public DriverOptions SeleniumOptions { get; set; }
+        public Browser BrowserName { get; set; } = Browser.Chrome;
+        public static List<string> w3cOptions = new List<string>(new string[]
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Change the visibility of 'w3cOptions' or make it 'const' or 'readonly'.](https://app.codacy.com/manual/SauceLabs/simple_sauce/pullRequest?prid=4945851)